apps/processor/src/index.ts

```diff
@@ -0,0 +1,33 @@
+import { inspect } from "util";
+
+import { environment } from "./config/index.js";
+import { ProcessorService } from "./services/processor.service.js";
+
+let processor: ProcessorService;
+
+const main = async (): Promise<void> => {
+    processor = new ProcessorService(environment);
```

Comment:
What about using `Processing` instead of `Processor` to avoid confusion with the event processors, if we modify this we should include modifying file names also

Reply:
aaa okay makes sense, Processor was a more natural naming. the app folder should also be `apps/processing`?